packages/bash/build.sh

```diff
@@ -1,20 +1,21 @@
 TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/bash/
 TERMUX_PKG_DESCRIPTION="A sh-compatible shell that incorporates useful features from the Korn shell (ksh) and C shell (csh)"
-TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_DEPENDS="ncurses, readline, libandroid-support, termux-tools, command-not-found"
-_MAIN_VERSION=4.4
-_PATCH_VERSION=23
-TERMUX_PKG_REVISION=4
-TERMUX_PKG_SHA256=d86b3392c1202e8ff5a423b302e6284db7f8f435ea9f39b5b1b20fd3ac36dfcb
+_MAIN_VERSION=5.0
+_PATCH_VERSION=2
+TERMUX_PKG_REVISION=5
```

Comment:
You can remove TERMUX_PKG_REVISION since TERMUX_PKG_VERSION is being bumped in this PR.